src/app/tests/suites/TestCluster.yaml

```diff
@@ -1317,6 +1317,17 @@ tests:
                         h: 0.1,
                     }
 
+    - label: "Send MEI command in cluster"
```

Comment:
Why put this and the attribute write both here and in TestUnitTestingClusterMei.yaml?  Not necessarily a problem, but seems redundant.

Reply:
To get some coverage in what is in CI